src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/ConfigBackup.xml

```diff
@@ -0,0 +1,176 @@
+﻿<configuration>
+  <!-- 
+    This is a copy of the old app.config for reference.
+    My intention is to replace all the XML config with code.
```

Comment:
I don't entirely follow what is proposed here.  @dmetzgar can you describe why the config info is being shrunk and why the "backup" config file is being populated?